bespin_lando_worker/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+lando_worker:
+  lando_version: 0.9.3
+  cwltool_version: 1.0.20180912090223
+  work_directory: /work
+  path_to_lando_worker: /usr/local/bin/lando_worker
```

Comment:
While I agree the path to the lando worker command is "magic", I'm not sure there's a use case for exposing it to be changed.

I believe the location of this file depends on where `pip3` puts it when it installs lando. Perhaps the better way to deal with the magic is to document that in the role, and maybe register the output of `which lando_worker` to get the absolute path for injecting into the startup script?